src/main/features/response/routes/claim-details.ts

```diff
@@ -4,18 +4,23 @@ import { Claim } from 'claims/models/claim'
 import { getInterestDetails } from 'common/interest'
 import { MoneyConverter } from 'fees/moneyConverter'
 import { MomentFactory } from 'common/momentFactory'
+import { Moment } from 'moment'
+import { InterestData } from 'app/common/InterestData'
 
 /* tslint:disable:no-default-export */
 export default express.Router()
   .get(Paths.claimDetailsPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
     try {
       const claim: Claim = res.locals.claim
       const feeAmount: number = MoneyConverter.convertPenniesToPounds(claim.claimData.feeAmountInPennies)
-      const interestData = await getInterestDetails(claim)
+      const interestData: InterestData = await getInterestDetails(claim)
+      const interestToDate: Moment = MomentFactory.currentDate().startOf('day').diff(interestData.interestFromDate, 'days') < 0 ?
```

Comment:
The same here `.startOf('day')` is not needed.

Reply:
What about moving this to `getInterestDetails` method so logic of calculating from and to date is kept close especially that the logic is more complex now.